eng/platform-matrix.yml

```diff
@@ -48,6 +48,7 @@ jobs:
     osIdentifier: Linux_musl
     containerName: musl_x64_build_image
     # TODO: add Alpine.Amd64 queues
+    portableBuild: false
```

Comment:
Alpine build was changed in corefx to portable (I've just merged in a PR from two weeks ago that was accidentally left unmerged https://github.com/dotnet/corefx/pull/34352). Is this file common for all of our distros? 
Btw, we should change Alpine coreclr build to portable too so that we can run on MUSL based distros that use different version of ICU libraries.

Reply:
The yaml file is coreclr specific. 
I will change musl to portable build in this PR. Thanks for pointing this out!